docker-compose/run_selenium_tests_remote.sh

```diff
@@ -90,6 +94,7 @@ then
             ;;
         TEST)
             export HOSTNAME_URL="https://test.bluebutton.cms.gov/"
+            export USE_LOGIN_WITH_MEDICARE_BUTTON='true'
```

Comment:
I think we might want delete this line and have this only determined by the `-l` setting. If test.medicare ends up reverting anything and getting rid of that button, then having this here means we won't be able to adjust settings for the tests to succeed. Maybe we could default to true for TEST, but have a way to override that to false if needed?